packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -5704,6 +5737,158 @@ class _ScribbleFocusableState extends State<_ScribbleFocusable> implements Scrib
   }
 }
 
+// TODO(justinmc): Some vertical handwriting seems to want to change the
+// selection instead of writing...
+class _Scribe extends StatefulWidget {
+  const _Scribe({
+    required this.child,
+    required this.editableKey,
+    required this.focusNode,
+  });
+
+  final Widget child;
+  final GlobalKey editableKey;
+  final FocusNode focusNode;
+
+  @override
+  State<_Scribe> createState() => _ScribeState();
+}
+
+class _ScribeState extends State<_Scribe> implements ScribeClient {
+  // The handwriting bounds padding of EditText in Android API 34.
+  static const EdgeInsets _handwritingPadding = EdgeInsets.symmetric(
+    horizontal: 10.0,
+    vertical: 40.0,
+  );
+
+  RenderEditable get _renderEditable => widget.editableKey.currentContext!.findRenderObject()! as RenderEditable;
+
+  @override
+  void initState() {
+    super.initState();
+    Scribe.registerScribeClient(this);
+  }
+
+  @override
+  void dispose() {
+    Scribe.unregisterScribeClient(this);
+    super.dispose();
+  }
+
+  // Begin ScribeClient.
+
+  @override
+  double get devicePixelRatio => MediaQuery.devicePixelRatioOf(context);
+
+  // TODO(justinmc): ScribbleClient does this in EditableText, setting the
+  // active client on Scribble. Maybe that's better? Reconcile?
+  @override
+  bool get isActive => widget.focusNode.hasFocus;
+
+  @override
+  bool performSelectionGesture(Rect selectionArea) {
+    // TODO(justinmc): Works, but selects even if only a tiny corner of a
+    // character is covered. I think by Android's definition, should have to
+    // cover the center of the granularity.
+    _renderEditable.selectPositionAt(
+      from: selectionArea.topLeft,
+      to: selectionArea.bottomRight,
+      // TODO(justinmc): Should this cause be generic or should there also be a scribe value?
+      cause: SelectionChangedCause.scribble,
+    );
+    return true;
+  }
+
+  // End ScribeClient.
+
+  void _handlePanDown(DragDownDetails details) {
+  }
+
+  void _handlePointerDown(PointerDownEvent event) {
+    print('justin _handlePointerDown.');
+    if (event.kind != ui.PointerDeviceKind.stylus) {
+      return;
+    }
+
+    if (!widget.focusNode.hasFocus) {
+      // TODO(justinmc): But don't show the keyboard!
+      widget.focusNode.requestFocus();
+    }
+    Scribe.startStylusHandwriting();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Listener(
```

Comment:
Note to self: I can listen directly to the global pointer binding to avoid the problem with clipping out events that I want.